### DISCUSSION TITLE
prConcurrentLimit default value is incorrect

### DISCUSSION THREAD

**Discussion by dudicoco** [renovatebot/renovate#14892] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

According to the docs, by default renovate does not limit the number of open concurrent PRs: https://docs.renovatebot.com/configuration-options/#prconcurrentlimit
> Limit to a maximum of x concurrent branches/PRs. 0 (default) means no limit.

However, I can see the following from the renovate logs: `DEBUG: Calculating prConcurrentLimit (10)`

Renovate indeed can only create 10 PRs in the repository.

Config file:
```json

   
{
  "extends": [
    "config:base"
  ],
  "labels": [
    "dependencies"
  ],
  "branchPrefix": "renovate_",
  "branchConcurrentLimit": 100,
  "rebaseWhen": "conflicted",
  "regexManagers": [
    {
      "fileMatch": [
        "(^|/)\\.pre-commit-config\\.yaml$"
      ],
      "matchStrings": [
        "\\nminimum_pre_commit_version: (?<currentValue>.*?)\\n"
      ],
      "depNameTemplate": "pre-commit",
      "datasourceTemplate": "pypi"
    },
    {
      "fileMatch": [
        "(^|/)\\.pre-commit-config\\.yaml$"
      ],
      "matchStrings": [
        "\\n\\s*entry: (?<depName>[^:]+):(?<currentValue>\\S+)"
      ],
      "datasourceTemplate": "docker"
    }
  ],
  "pre-commit": {
    "enabled": true
  }
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
the `config:base` preset overrides default renovate settings with recommend values. check the docs about it.

https://docs.renovatebot.com/presets-config/#configbase

**Reply by dudicoco** [reply]
Thanks @viceice, forgot about `config:base` :)
